build.sh

```diff
@@ -29,7 +29,8 @@ function FirstJsonValue {
 
 # If dotnet CLI is installed globally and it matches requested version, use for execution
 if [ -x "$(command -v dotnet)" ] && dotnet --version &>/dev/null; then
-    export DOTNET_EXE="$(command -v dotnet)"
+    DOTNET_EXE="$(command -v dotnet)"
```

Comment:
should we really change build.sh here?

Reply:
... seems in OTel repo we did 🤷

Reply:
It is easier to change. John is trying to propagate changes to upstream.